app/models/api/openid_connect/authorization.rb

```diff
@@ -0,0 +1,79 @@
+module Api
+  module OpenidConnect
+    class Authorization < ActiveRecord::Base
+      belongs_to :user
+      belongs_to :o_auth_application
+
+      validates :user, presence: true
+      validates :o_auth_application, presence: true
+      validates :user, uniqueness: {scope: :o_auth_application}
```

Comment:
This can be joined with the validation on line 7, the point of `validates` is that you can specify multiple for the same attribute.